spec/features/residents_can_see_confirmation_of_repair_request_spec.rb

```diff
@@ -2,8 +2,38 @@
 
 RSpec.feature 'Resident can see a confirmation of their repair request' do
   scenario 'when the repair request creation was successful' do
-    visit '/confirmations/00012143'
+    property = {
+      'property_reference' => '00000503',
+      'short_address' => 'Ross Court 30',
+      'postcode' => 'E5 8TE',
```

Comment:
Is the API `list_properties` endpoint being changed to return the postcode in the results, because it doesn't at present..?

Reply:
Yes: https://trello.com/c/VNIlJMI7/113-the-properties-propertyref-endpoint-should-return-postcode

I'm assuming this endpoint will ultimately return significantly more data - it was only this minimal as a first stab.

I considered merging the postcode into the answer hash ourselves, but we'd have to pass it from the radio buttons, which would get quite funky.